src/App.tsx

```diff
@@ -1,15 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
 export const App: React.FC = () => {
-  const count = 0;
+  const [count, setCount] = useState(0);
 
   const addOne = () => {
-    // increase count by 1
+    setCount(newCount => newCount + 1);
```

Comment:
it's rather a prevCount
```suggestion
    setCount(prevCount => prevCount + 1);
```